save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt

```diff
@@ -28,6 +32,12 @@ class Vulnerability(
     @JsonIgnore
     var projects: List<VulnerabilityProject>,
 
+    var isActive: Boolean,
+
+    var createDate: LocalDateTime,
```

Comment:
maybe make population of these fields via `EntityListener`?
It can be re-used in future in another entities

Reply:
Added